src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+min_read: 5
+date: 2020-02-15
+---
+
+### Wat is Phaser?
+Kort gezegd is [Phaser](https://phaser.io/) een gameframework waarmee je HTML-5 games kan maken. Deze games kunnen gespeeld worden op desktop en mobiel. Phaser kan gratis gedownload worden via https://phaser.io/ en is ontwikkeld door Photon Storm in 2013.
+
+Om gebruik te maken van Phaser is het handig als je al enige kennis hebt van HTML en JavaScript. Phaser is namelijk een JavaScript library, wat dus betekent dat je de game volledig hierin bouwt. Phaser maakt het mogelijk voor beginners en gevorderden om een eigen interactieve en leuke game te maken. Verder heeft het veel verschillende functies, zelfs te veel om allemaal te benoemen. Dit is ook een van de redenen waarom het zo geliefd is bij game ontwikkelaars. 
+
+### Opzetten van een Phaser game
+Het begint allemaal bij je HTML bestand waar je Phaser aanroept in je head, met de volgende code: 
+```
+<script src="https://cdnjs.cloudflare.com/ajax/libs/phaser-ce/2.12.0/phaser.min.js"></script>
+```
+Na dit gedaan te hebben maak je een JavaScript bestand aan en roep je de game aan met:
+```
+window.onload = function(){
+    var game = new Phaser.Game(800 (width), 600 (height), Phaser.AUTO, "", {
+        preload: preload,
+        create: create,
+        update: update
+  }); 
+```
+Als weergavecontext voor de game kan er gekozen worden voor Phaser.CANVAS, Phaser.WEBGL of Phaser.AUTO. De aanbevolen waarde om te gebruiken is Phaser.AUTO, omdat deze automatisch dat gebruikt wat de browser die wordt gebruikt ondersteunt. 
+
+Wanneer je toch wil kiezen voor WEBGL of CANVAS is het goed om de verschillen te weten. Een van de verschillen is dat WEBGL qua snelheid groter is dan CANVAS. Daarnaast wordt WEBGL door meer browsers ondersteunt en is vooral bedoelt voor 3D, maar kan ook gebruikt worden voor 2D. Daarentegen kan CANVAS alleen gebruikt worden voor 2D weergave. CANVAS is dan ook de voorloper van WEBGL, want WEBGL is voortgekomen uit CANVAS 3 experimenten.
```

Comment:
_Qua snelheid groter is_ -> Hoe bedoel je dit precies? is WEBGL groter que formaat dus is het langzamer of is het juist sneller

Reply:
WEBGL is sneller dan dan CANVAS. Heb het nu veranderd in : `Een van de verschillen is dat WEBGL een stuk sneller is dan CANVAS. `